masterlist.yaml

```diff
@@ -22544,8 +22850,6 @@ plugins:
         itm: 141
   - name: 'Alternate Start - Live Another Life.esp'
     global_priority: 70
-    tag:
-      - C.Location
```

Comment:
Same reason as in [#136 (comment)](https://github.com/loot/skyrim/pull/136#discussion_r122197248)